### DISCUSSION TITLE
[Feature Request] Add newer MAME releases (0.187)

### DISCUSSION THREAD

**Discussion by freq-mod** [OnionUI/Onion#543] [Ideas]
As long as it is possible, it will improve Neo Geo emulation accuracy.

**Comment by Olywa** [comment]
Mame is really not the best emulator for NeoGeo using the mini due to it's hardware limitations, a newer version of Mame is likely to struggle also as it will require more resources (and will have a detrimental effect to other working games).

OnionOS has an dedicated NeoGeo emulator which runs those games perfectly using the `fbalpha2012_neogeo_libretro.so` RA core. You have to ensure you are using the correct FBA rom versions (FBAlpha v0.2.97.24 or v0.2.97.29 or as close to these as you can find) and have a copy of the neogeo.zip (bios) in you Roms/NEOGEO folder with the games and also in the BIOS folder.

**Reply by freq-mod** [reply]
The reason I requested 0.187 is this homebrew demo: https://www.youtube.com/watch?v=WyxvDh57zZY

I know it works fine on EmuELEC devices (and RK3326 handheld, iirc). Final Burn Alpha  straight up refuses to load it, Final Burn Neo complians about neogeo.zip being missing which is clearly not the case, no matter where i put fbneo directory (although I can experiment with paths).

**Reply by Olywa** [reply]
Ah OK, well the RK3326 is more powerful, those devices also have a GPU and more memory so have much more available horsepower than the miyoo mini. It's not something to compare. The mini has no GPU and only 128mb Ram. 

I can tell you 100% that all NeoGeo games run perfectly and at full speed using the **default** NeoGeo emulator (fbalpha2012_neogeo_libretro) from the Consoles Menu (not Expert). If you are having issues, either your roms or bios is where the problem is. 
1) Make sure you Bios (NeoGeo.zip) is from a good source and is in BOTH the Bios folder (on the root of SD) and a copy is also in the same folder with your game roms.
2) Make sure your roms are matched (or closely matched) to the version I mentioned above (the roms that came with the mini are a mixed bag, not good)


FBNeo also expects more horsepower than the Mini has to offer, I don't recommend using it on this device.
